Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -199,6 +199,8 @@
     <string name="stereoscopy_convergence_description">Control the distance of the convergence plane, this is the distance at which objects will appear to be in front of the screen.\nA higher value creates stronger out-of-screen effects while a lower value is more comfortable.</string>
     <string name="stereoscopy_swap_eyes">Swap Eyes</string>
     <string name="stereoscopy_swap_eyes_description">Swap the left and right eye, mostly useful if you want to view side-by-side cross-eyed.</string>
+    <string name="backend_multithreading">Backend Multithreading</string>
+    <string name="backend_multithreading_description">Enables CPU multithreading(Vulkan only). May affect performance. If unsure, leave unchecked</string>
```

Comment:
do another copyediting pass on this, and leave a comment explaining that the default is different on Android and other platforms

Reply:
Added comment